Add documentation for completion (#1058)
shopware/devdocs: source/developers-guide/shopware-5-cli-commands/index.md

```

As you can see, a Shopware CLI command is very similar to a Symfony 3 command. You can use any of the features provided by the Symfony Console component, like you would in Symfony 3 itself.

## Add completion for commands

> Shopware uses [stecman/symfony-console-completion](https://github.com/stecman/symfony-console-completion) to add completion features you already know from any CLI.

Reviewer: ```suggestion
Shopware uses [stecman/symfony-console-completion](https://github.com/stecman/symfony-console-completion) to add completion features you might already know from other CLI.
```

> This composer package automatically adds completion to any command about the option names.

Reviewer: do you mean `... via the option names` ? otherwise I don't understand this sentence :blush:

Author: I wanted to say that by default this package can only expand option names. I think about a better sentence.

> This composer package automatically adds completion to any command about the option names.

Reviewer: ```suggestion
This composer package automatically adds completion for options to any command.
```
maybe like this?

Author: Yes, it is easier to understand.
I recognized I also need to mention command names itself. I mean `sw:pl` expands to `sw:plugin:` so this is also part of the package that should get mentioned.

Reviewer: just let me know, when I could review again :blush:

Author: Right here, right now :musical_note:

> Some option names need values like the `--batch` option from `sw:plugin:update` the package can't know the allowed values.

Reviewer: ```suggestion
Some option names need values like the `--batch` option from `sw:plugin:update`, so the package can not recognize the allowed values automatically.
```

or what do you want to say here?
